packages/console/src/pages/Security/Captcha/CreateCaptchaForm/constants.ts

```diff
@@ -73,14 +73,17 @@ export const captchaProviders: CaptchaProviderMetadata[] = [
       {
         field: 'siteKey',
         label: 'security.captcha_details.recaptcha_key_id',
+        placeholder: 'security.captcha_details.recaptcha_key_id',
```

Comment:
Are these phrases already committed?

Reply:
this is the same as the above one, no extra phrases